src/Objectivity.AutoFixture.XUnit2.Core/Attributes/ValuesAttribute.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Objectivity.AutoFixture.XUnit2.Core.Attributes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+
+    using global::AutoFixture;
+    using global::AutoFixture.Kernel;
+    using global::AutoFixture.Xunit2;
+
+    using Objectivity.AutoFixture.XUnit2.Core.Common;
+    using Objectivity.AutoFixture.XUnit2.Core.Requests;
+    using Objectivity.AutoFixture.XUnit2.Core.SpecimenBuilders;
+
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false)]
+    public sealed class ValuesAttribute : CustomizeAttribute
```

Comment:
It would be possible to create an attribute that would generate separate test case for each of provided values? I.e. by somehow using `InlineAutoMockData` in the background. Though it may be a bit tricky if we have multiple arguments with this attribute.
I think I'd use this test case generating attribute more often, because if these values are so "unique", we should be testing all of cases each time.